TestAll.ps1

```diff
@@ -0,0 +1,110 @@
+#Requires -RunAsAdministrator
+
+Param(
+    [string]$OutputFolder = "",
+    [string]$AzureBuildStep = "all",
+    [string]$Platform = "x64",
+    [string]$Configuration = "release",
+    [string]$ImageName = "",
+    [switch]$AzurePipelineBuild = $false,
+    [switch]$Help = $false
+)
+
+$StartTime = Get-Date
+
+if ($help) {
+  Write-Host @"
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+Syntax:
+      TestAll.cmd [options]
+
+Description:
+      Builds the WindowsAppSDK aggregate NuGet package.
+
+Switches:
+
+  -Platform <platform>
+      Only test the selected platform.
+      Example: -Platform x64
+
+  -Configuration <configuration>
+      Only build the selected configuration(s).
```

Comment:
Since this script is being used by both the build pipeline and local, I think both parameters should support lists. 

Platforms: "x64,x86"
Configuration:"release,debug"

But the pipeline will only submit the pair, not a list: x64 | release.

Reply:
For v1 I don't think we have to support lists. Will follow up with this as a feature.